specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -111,6 +110,10 @@ const processFieldType: {
       console.error("Can't render a check box for a relationship field");
       return { type: 'Blank' };
     }
+    const isSendEmail = name === 'sendEmail';
+    if (isSendEmail) {
+      return { type: 'Blank' };
+    }
```

Comment:
Could you include a comment stating the sendEmail button is not supported in Specify 7 and provide a link to the issue?

I am also not sure if the `isSendEmail` variable is needed. I personally think the condition itself (when paired with a comment) would make it clear enough

Reply:
I would imagine something like this would be sufficient

```js
/**
     * The send email button is not supported in Specify 7
     * See https://github.com/specify/specify7/issues/2211
     *  */
    if (name === 'sendEmail') {
      return { type: 'Blank' };
    }
```